xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,239 @@
+   </para>
+   <para>
+    This is the most common method for deploying operating systems on the
+    &rpi; platform.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Boot from USB media and install onto another drive.
+   </para>
+   <para>
+    This is the more conventional way to install &sls;, although it is less
+    common for &rpi; operating systems.
+   </para>
+  </listitem>
+ </itemizedlist>
+ 
+ <para>
+  There are several points to note regarding both methods.
+ </para>
+ 
+ <sect2>
+  <title>Deploying from a Preinstalled Image</title>
+  <para>
+   This method is the simplest and easiest method. However, it allows little
+   customisation of the product being deployed except for values such as network
+   and account settings.
+  </para>
+  <para>
+   &suse; provides a pre-configured bootable image of &sle; for &rpi; 3 series
+   hardware. This comes with the lightweight IceWM desktop and the Btrfs file
+   system, with compression enabled to improve performance and reduce wear on
+   MicroSD media.
+  </para>
+  <para>
+   A MicroSD card with a minimum size of 8 GB is recommended. Faster cards will
+   give better system performance. On the first boot, the operating system
+   automatically expands the filesystem to fill the card. This means that the
+   first boot will be substantially slower than subsequent boots.
+  </para>
+  <para>
+   &sle; 15.1 adds support for the &rpi; 3 Model B+, which was not fully
+   functional using &sle; 15. Apart from this, the installation process is the
+   same as it was for &sle; 12 SP3. This is documented here: 
+   <link xlink:href="https://www.suse.com/documentation/suse-best-practices/sles-for-arm-raspberry-pi/data/sles-for-arm-raspberry-pi.html"/> 
+  </para>
+  
+ </sect2>
+ <sect2>
+  <title>Installation from USB Media</title>
+  <para>
+   This method will be more familiar as it is the way &sls; is deployed
+   on other platforms. Installation can be performed from a removable USB
+   medium, such as a memory stick, onto a MicroSD card in the machine's internal
+   slot, or from a removable USB medium onto another USB medium, such as a
+   USB-connected hard disk.
+  </para>
+  <note>
+   <title>USB Bandwidth Limitations</title>
+   <para>
+    Note that the Ethernet, Wi-Fi and Bluetooth controllers on the &rpi; are all
+    connected to the device's on-board USB 2 bus. Therefore an operating system
+    running from a disk attached via USB must share the total 480 MBps bandwidth
+    of the USB controller. This will limit performance, and could significantly
+    impact network performance.
+   </para>
+  </note>
+  <para>
+   Installation on the Raspberry Pi has some aspects that are unlike other
+   platforms.
+  </para>
+  <para>
+   The &rpi; differs from more conventional machines in two important ways:
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     The &rpi; hardware does not have any built-in firmware. Its firmware is
```

Comment:
Make that &rpi; 3 please, in preparation for 4.

Reply:
I can do that but as far as I know the RPi 4 also has no built-in firmware. It still loads the firmware image from SD card, does it not?

Reply:
Replaced in the § about USB bandwidth as this does not apply to the RPi 4.